.gitignore

```diff
@@ -264,7 +264,3 @@ cython_debug/
 .ionide
 
 # End of https://www.toptal.com/developers/gitignore/api/python,visualstudiocode,pycharm
-beam.dat
-detect.dat
-geo.dat
-mat.dat
```

Comment:
Did these lines serve any purpose? Shouldn't we have a similar rule?

Maybe it should be forbidden to commit *.dat files except in some locations?
Something like:

```.gitignore
**/*.dat
!input_examples/expected_shieldhit_output/*.dat
```

or alternatively, we can globally ignore all `*.dat` files 
```.gitignore
**/*.dat
```
and add an additional `.gitignore` file in `input_examples/expected_shieldhit_output`
```.gitignore
!*.dat
```

Reply:
Done in 089ad72ab7919d792fcfbef864cf85543abf2363